src/libraries/System.Net.Http/tests/StressTests/HttpStress/Dockerfile

```diff
@@ -7,13 +7,13 @@ RUN apt-get update -y && \
     apt-get upgrade -y && \
     apt-get install -y cmake clang ruby-dev gem lttng-tools libssl-dev && \
     gem install fpm
-RUN git clone --recursive https://github.com/dotnet/msquic
-RUN cd msquic/src/msquic && \
+RUN git clone --recursive https://github.com/microsoft/msquic --depth 1
+RUN cd msquic/ && \
     mkdir build && \
-    cmake -B build -DCMAKE_BUILD_TYPE=Release -DQUIC_ENABLE_LOGGING=false -DQUIC_USE_SYSTEM_LIBCRYPTO=true -DQUIC_BUILD_TOOLS=off -DQUIC_BUILD_TEST=off -DQUIC_BUILD_PERF=off -DQUIC_TLS=openssl3 && \
+    cmake -B build -DCMAKE_BUILD_TYPE=Debug -DQUIC_ENABLE_LOGGING=false -DQUIC_USE_SYSTEM_LIBCRYPTO=true -DQUIC_BUILD_TOOLS=off -DQUIC_BUILD_TEST=off -DQUIC_BUILD_PERF=off -DQUIC_TLS=openssl3  -DQUIC_ENABLE_SANITIZERS=on && \
```

Comment:
> Should we also use debug build of msquic as it has more asserts and sanity checks?

Doesn't change the behavior way too much? Can we compare the total number of stress RPS main vs PR in a stable environment to have a rough feeling about the difference and see how important it is to have different runs set up?

If we think that running msquic in debug is justified because of additional checks, the same reasoning might also justify running stress against debug builds of libraries code.

Reply:
I would think that what perf tests would be for. In my mind the stress is just to shake out the tree. I'm certainly fine if we don't do it right now.

Reply:
> running stress against debug builds of libraries code.

That won't fly until https://github.com/dotnet/runtime/issues/93713 is fixed.